onnxscript/_framework_apis/torch_2_5.py

```diff
@@ -68,32 +67,17 @@ def save_model_with_external_data(model: ir.Model, model_path: str | os.PathLike
     """Save the model with external data. The model is unchanged after saving."""
 
     # TODO(#1835): Decide if we want to externalize large attributes as well
-    initializer_values = tuple(model.graph.initializers.values())
-    tensors = [v.const_value for v in initializer_values]
+    tensors = [v.const_value for v in model.graph.initializers.values()]
     for tensor in tensors:
```

Comment:
nit: if this is the only use of tensors here, we can just rewrite as
`
for v in model.graph.initializers.values():
if v.const_value is None:
`

Reply:
Done, thanks!